src/page-modules/assistant/trip/trip-pattern/index.tsx

```diff
@@ -108,9 +108,30 @@ export default function TripPattern({
                     )}
                   </div>
 
-                  <Typo.span textType="body__tertiary">
-                    {formatToClock(leg.expectedStartTime, language, 'floor')}
-                  </Typo.span>
+                  <div className={style.timeStartContainer}>
+                    {secondsBetween(leg.aimedStartTime, leg.expectedStartTime) >
+                    60 ? (
```

Comment:
Could add the `60` as a const at the top of the file 👍 

```suggestion
                    DEFAULT_THRESHOLD_AIMED_EXPECTED_IN_SECONDS ? (
```